rust/sealevel/README.md

```diff
@@ -0,0 +1,127 @@
+# Hyperlane Sealevel (Solana VM) Integration
+
+# Running local end to end test
+
+A local end to end test has been written that will:
+
+1. Run a local Solana network
```

Comment:
Steps 1 and 5 still need to be performed manually, I think this should be automated so the github actions run the e2e test. High-prio IMO but could be a subsequent PR

Reply:
agree - this will be done in https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/2415